### PR TITLE
Test to verify that initramfs does not have injected configuration

### PR DESCRIPTION
**Pull request by gsr-shanks**

### CODE REVIEW

**Comment by gsr-shanks** [comment]
@dshchedr

File: functests/1_performance/performance.go [hunk 1]
```diff
@@ -116,6 +116,26 @@ var _ = Describe("[rfe_id:27368][performance]", func() {
...
+				found := false
+				imagesPath := strings.Split(string(initramfsImagesPath), "\n")
+				for _, imagePath := range imagesPath[:2] {
+					initrd, err := nodes.ExecCommandOnMachineConfigDaemon(&node,
+						[]string{"lsinitrd", strings.TrimSpace(imagePath)})
+					Expect(err).ToNot(HaveOccurred())
+					initrdString := string(initrd)
+					if strings.Contains(initrdString, "'/etc/systemd/system.conf /etc/systemd/system.conf.d/setAffinity.conf'") {
+						found = true
+						break
```
Comment (yanirq) [inline functests/1_performance/performance.go:132]: we break here once we found the string but the image is not necessarily the right one .
`find", "/rootfs/boot/ostree/", "-name", "*.img"`  can give several images of the image phases

File: functests/1_performance/performance.go [hunk 2]
```diff
@@ -116,6 +117,21 @@ var _ = Describe("[rfe_id:27368][performance]", func() {
 			}
 		})
 
+		It("[test_id:32375][crit:high][vendor:cnf-qe@redhat.com][level:acceptance] initramfs should not have injected configuration", func() {
+			for _, node := range workerRTNodes {
+				rhcosId, err := nodes.ExecCommandOnMachineConfigDaemon(&node, []string{"awk", "-F", "/", "{printf $3}", "/rootfs/proc/cmdline"})
+				Expect(err).ToNot(HaveOccurred())
+				initramfsImagesPath, err := nodes.ExecCommandOnMachineConfigDaemon(&node, []string{"find", filepath.Join("/rootfs/boot/ostree", string(rhcosId)), "-name", "*.img"})
+				Expect(err).ToNot(HaveOccurred())
+				found := false
```
Comment (cynepco3hahue) [inline functests/1_performance/performance.go:126]: hm can you change it to
```
Expect(string(initramfsImagesPath)).Should(ContainSubstring("'/etc/systemd/system.conf /etc/systemd/system.conf.d/setAffinity.conf'"))
```

**Comment by cynepco3hahue** [comment]
@yanirq Can you please review again?

**Pull request merged**